src/backup_camera.h

```diff
@@ -22,7 +22,7 @@ extern "C" {
 #ifndef RUNNINGONPI
 #define RUNNINGONPI
 #endif
-#include "WiringPiButtons.hpp"
+#include <wiringPi.h>
```

Comment:
Uhh I guess I merged a bit early, please look into this still. wiring pi was when we had GPIOs connected to the pi, but we don't need that anymore. Pretty sure we can delete all traces of it.